### PR TITLE
Feature/2577 deployment success failure msg

### PR DESCRIPTION
**Pull request by justinlevi**
Removes S3 logging

### CODE REVIEW

File: services/openshiftbuilddeploymonitor/src/index.js [hunk 1]
```diff
@@ -3,15 +3,16 @@
 const Promise = require("bluebird");
 const OpenShiftClient = require('openshift-client');
 const sleep = require("es7-sleep");
-const AWS = require('aws-sdk');
-const uuidv4 = require('uuid/v4');
+// const AWS = require('aws-sdk');
+// const uuidv4 = require('uuid/v4');
```
Comment (Schnitzel) [inline services/openshiftbuilddeploymonitor/src/index.js]: let's remove this

File: services/kubernetesbuilddeploymonitor/src/index.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 const promisify = require('util').promisify;
 const kubernetesClient = require('kubernetes-client');
 const sleep = require("es7-sleep");
-const AWS = require('aws-sdk');
-const uuidv4 = require('uuid/v4');
+// const AWS = require('aws-sdk');
+// const uuidv4 = require('uuid/v4');
```
Comment (Schnitzel) [inline services/kubernetesbuilddeploymonitor/src/index.js]: let's remove please

File: services/api/src/resources/deployment/resolvers.js [hunk 3]
```diff
@@ -166,6 +166,26 @@ const getDeploymentByRemoteId = async (
...
 };
 
+const getDeploymentUrl = async (
+  { id, environment },
+  args,
+  { sqlClient, hasPermission },
+) => {
+
+
+  const domain = process.env.LAGOON_UI || 'localhost';
```
Comment (Schnitzel) [inline services/api/src/resources/deployment/resolvers.js]: we have a system in the api already that allows Lagoon to autodiscover the URLs of other services, see
https://github.com/amazeeio/lagoon/blob/1386194da63bd267c47a3534a3d375fe474caa3b/services/api/src/clients/keycloakClient.js#L21-L26

Can you implement this here?

**Review by Schnitzel — CHANGES_REQUESTED** [review]
yay nice work @justinlevi 
just a couple of small things

**Comment by justinlevi** [comment]
All requested fixes should be done.

**Pull request merged**